Dockerfile

```diff
@@ -91,6 +91,12 @@ RUN apt-get update -q && \
     phpdismod pdo_pgsql && \
     phpdismod pgsql && \
     phpdismod xdebug && \
+    # Remove extra extensions installed via packages for other versions of PHP
+    rm -rf /usr/lib/php/20121212 && \
```

Comment:
since extensions are added by package, an `.so` file is added per php version, whether they are installed or not. removing...